Zend/zend_API.h

```diff
@@ -1835,6 +1850,39 @@ static zend_always_inline int zend_parse_arg_str_or_array_ht(
 	return 1;
 }
 
+static zend_always_inline int zend_parse_arg_str_or_long_slow(zval *arg, zend_string **dest_str, zend_long *dest_long)
```

Comment:
This should not be zend_always_inline. It should be in the C file. It should also only check for strict types once and then use _weak rather than _slow.

Reply:
Apologies I didn't total understand what you meant the first time around, should have fixed that now.